README.md

```diff
@@ -24,7 +24,9 @@ Implementation status (or even intention) isn't tracked here; see [Firefox Platf
 The currently possible positions are:
 
 - `under consideration` - Mozilla's position on this specification is being discussed.
```

Comment:
This might belong between `non-harmful` and `defer`, so that the positions are sorted by how favorable Mozilla is.

Reply:
I think I wanted to keep the current status of having `under consideration` at the start, since it's actually *not* a position.

Reply:
Fair enough.